integration/specs/accordion-1.spec.js

```diff
@@ -0,0 +1,141 @@
+const PageAccordion = require('./../../src/components/Accordion/pageObject');
+const {
+    SPACE_KEY,
+    ENTER_KEY,
+    ARROW_DOWN_KEY,
+    ARROW_UP_KEY,
+    ARROW_LEFT_KEY,
+    ARROW_RIGHT_KEY,
+    TAB_KEY,
+} = require('./../constants');
+
+const ACCORDION = '#accordion-1';
+
+describe('Accordion whend multiple is not passed', () => {
+    beforeEach(() => {
+        browser.url('/#!/Accordion/1');
+        browser.refresh();
+    });
+    it('should expand the accordion section when the accordion section is collapsed and click on the button icon', () => {
+        const accordion = new PageAccordion(ACCORDION);
+        const accordionSection = accordion.getItem(0);
+        accordionSection.clickButtonIcon();
+        expect(accordionSection.isExpanded()).toBe(true);
+    });
+    it('should collapse the accordion section when click twice on the button icon', () => {
+        const accordion = new PageAccordion(ACCORDION);
+        const accordionSection = accordion.getItem(0);
+        accordionSection.clickButtonIcon();
+        accordionSection.clickButtonIcon();
+        expect(accordionSection.isExpanded()).toBe(false);
+    });
+    it('should close the firstaccordoin section and expand the second accordion section when the first accordion section is expanded and click on the second button icon', () => {
```

Comment:
should close the first accordion section and expand the second when the first is expanded and is clicked the button icon of the second